install-server.sh

```diff
@@ -71,6 +71,12 @@ check_deps() {
         echo "unzip is required by this script, please install it and try again."
         exit 1
     fi
+    if grep -q "Y" /sys/module/apparmor/parameters/enabled; then
+        if ! which apparmor_parser >/dev/null 1>&1; then
```

Comment:
nit: consider using the `command` built-in instead of `which` (ref: https://stackoverflow.com/questions/37056192/which-vs-command-v-in-bash).